modules/beamdyn/src/Driver_Beam.f90

```diff
@@ -98,8 +98,7 @@ PROGRAM BeamDyn_Driver_Program
       
       ! initialize the BD_InitInput values not in the driver input file
    BD_InitInput%RootName = TRIM(BD_Initinput%InputFile)
-   BD_InitInput%RootDisp = matmul(transpose(BD_InitInput%RootOri),BD_InitInput%GlbPos) - BD_InitInput%GlbPos
-   BD_InitInput%RootVel(1:3) = matmul(BD_InitInput%RootOri, Cross_Product( BD_InitInput%RootVel(4:6), BD_InitInput%GlbPos ))  ! set translational velocities based on rotation and GlbPos.
+   BD_InitInput%RootDisp = 0.d0
```

Comment:
With this change, how do you simulate starting the blade at a different azimuth?

Reply:
The displacement is relative to its initial position.   Part of my verification was to show that results were the same as those produced by openfast at the different azimuth.